packages/upset/src/components/Columns/Attribute/AttributeBar.tsx

```diff
@@ -11,10 +11,11 @@ import translate from '../../../utils/transform';
 import { BoxPlot } from './AttributePlots/BoxPlot';
 import { DotPlot } from './AttributePlots/DotPlot';
 import { itemsAtom } from '../../../atoms/itemsAtoms';
+import { DeviationBar } from '../DeviationBar';
 
 type Props = {
```

Comment:
I think documentation would be useful for this type as well, particularly as to why `summary` can be a `FiveNumberSummary` or just a single `number`